tests/testthat/test-error.R

```diff
@@ -42,4 +42,32 @@ test_that("returns the correct linting", {
   expect_lint("\\",
     rex("unexpected input")
   )
+
+  expect_lint("``",
+    rex("attempt to use zero-length variable name")
+  )
+
+  expect_lint("``()",
+    rex("attempt to use zero-length variable name")
+  )
+
+  expect_lint("''()",
+    rex("attempt to use zero-length variable name")
+  )
+
+  expect_lint("\"\"()",
```

Comment:
nit: `'""'` was used in the source (and IMO is easier on the eyes), I would use that here too (e.g. `'""()'`)

Reply:
So we don't have a policy to only allow one type of quotes? Then sure it is better.

Reply:
Somehow I thought the tidyverse rule was "use double quotes, unless using single quotes to avoid escaping single quotes", but I don't see it in the style guide actually.

However, `single_quotes_linter` in this package makes that distinction :smiley: 

https://github.com/jimhester/lintr/blob/master/R/single_quotes_linter.R#L8

Reply:
Thanks!